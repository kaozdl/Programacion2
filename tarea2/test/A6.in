insertar_lista (1,tigre) (2,bufalo) (3,conejo) .
reversa_lista 1 3
modificar_lista 2 3
reversa_lista 3 2 
liberar_lista
Fin
