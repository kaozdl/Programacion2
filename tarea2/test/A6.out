1>(0,primero)(1,tigre)(2,bufalo)(3,conejo)
2>(1,tigre)(2,bufalo)(3,conejo)
(3,conejo)(2,bufalo)(1,tigre)
(3,conejo)(2,bufalo)(3,tigre)
3>(0,primero)(1,tigre)(3,conejo)(2,bufalo)
(0,primero)(1,tigre)(2,bufalo)(3,conejo)
4>5>6>Fin.
